### DISCUSSION TITLE
How to display label names?

### DISCUSSION THREAD

**Discussion by vinot** [open-mmlab/mmpose#1164] [Q&A] [answered]
how to display name of labels in output video? for boxes and keypoints

ie:
https://user-images.githubusercontent.com/607087/151859751-979d0241-bb70-4585-9500-445f4ff15cd8.mp4

Thank you

**Comment by jin-s13** [comment, marked as answer]
Pls check this function.

https://github.com/open-mmlab/mmpose/blob/e22b871f0675cdfdc35e0385db13dbfd4a9f1bcf/mmpose/core/visualization/image.py#L149

Add cv2.putText() as you like

**Comment by vinot** [comment]
thank you!
